deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -311,6 +311,23 @@ def get_base_images(request):
     })
 
 
+def add_acceptance(request, base_images):
```

Comment:
this looks like a helper method and should not be visible outside of class?
Also, might want to rename it to reflect it's not updating any record ex: populate_acceptance_status

Reply:
actually I think this can be part of baseimages_helper

Reply:
Good idea, moved.